_config/email.yml

```diff
@@ -1,12 +1,55 @@
 ---
-Name: emailconfig
+Name: mailer
 ---
 SilverStripe\Core\Injector\Injector:
-  Swift_Transport: Swift_MailTransport
-  Swift_Mailer:
-    constructor:
-      - '%$Swift_Transport'
-  SilverStripe\Control\Email\Mailer:
-    class: SilverStripe\Control\Email\SwiftMailer
-    properties:
-      SwiftMailer: '%$Swift_Mailer'
+  Symfony\Component\Mailer\MailerInterface:
+    class: Symfony\Component\Mailer\Mailer
+    constructor:
+      0: '%$Symfony\Component\Mailer\Transport\TransportInterface'
```

Comment:
I think I'd prefer using the actual argument names instead of array indices if we're going to add keys for these. May as well be explicit about what's being set.

Reply:
Updated